llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -553,6 +557,50 @@ class Instruction : public sandboxir::User {
 #endif
 };
 
+class LoadInst : public Instruction {
```

Comment:
Is `LoadInst` final?

Reply:
yes

Reply:
You can mark `LoadInst` as final for documentation and optimization.

Reply:
Done